connect/runtime/src/test/java/org/apache/kafka/connect/util/clusters/EmbeddedConnectCluster.java

```diff
@@ -126,11 +154,46 @@ public void startConnect() {
         putIfAbsent(workerProps, KEY_CONVERTER_CLASS_CONFIG, "org.apache.kafka.connect.storage.StringConverter");
         putIfAbsent(workerProps, VALUE_CONVERTER_CLASS_CONFIG, "org.apache.kafka.connect.storage.StringConverter");
 
-        for (int i = 0; i < connectCluster.length; i++) {
-            connectCluster[i] = new ConnectDistributed().startConnect(workerProps);
+        for (int i = 0; i < numInitialWorkers; i++) {
+            connectCluster.add(new ConnectDistributed().startConnect(workerProps));
         }
     }
 
+    /**
+     * Get the workers that are up and running.
```

Comment:
Nit: this returns the _URLs_ of the online workers in this cluster. Would be good to state this explicitly in the JavaDoc, including in the `@return` statement

Reply:
done